sdk/storage/storage-file/src/DirectoryClient.ts

```diff
@@ -56,6 +56,17 @@ export interface DirectorySetMetadataOptions {
   abortSignal?: Aborter;
 }
 
+interface AzureDirectoryItem {
```

Comment:
Export `AzureDirectoryItem` and `AzureDirectoryItem` here and in `index.ts` and `index.browser.ts` so users can consume them.

Reply:
Done